### ISSUE TITLE
Docking fails for cinnamaldehyde in OR3A1.

### ISSUE THREAD

**Issue by ssepeq** [primaryodors/podock#85]
Tried to dock OR3A1 with cinnamaldehyde just now and no poses resulted.

Investigate why this is happening and fix it. Cinnamaldehyde is known to be a 3A1 agonist.

**Comment by ssepeq** [comment]
OR3A1 is known to respond to aldehydes, but fails to respond if the aldehyde moiety is replaced by an alcohol, ketone, vinyl, or acetate. Further, a 2-carbon chain must separate the aldehyde moiety from the aromatic ring, and there should be a methyl group attached to either the α- or β- carbon. If the carbon chain contains a double bond, it decreases the activity of the ligand.

From here: https://onlinelibrary.wiley.com/doi/10.1111/j.1471-4159.2006.03771.x

Due to the structural features identified as important for 3A1 agonism, one can expect that the shared features of the different molecules should tend to dock into basically the same positions of the receptor, and interact with the same residues each time. This is not happening with e.g. helional vs. lilial, but so far the sample size is too small.

Note that these criteria exclude cinnamaldehyde because of its double bond and its lack of a methyl group. However, [Mainland et al](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4412152/) do identify cinnamaldehyde as a 3A1 agonist. From a sensory perspective, 3A1 agonists other than cinnamaldehyde are universally characterized by either a green-vegetable note or a vanilla note, so the inclusion of cinnamaldehyde in the list seems suspect. The criteria do, however, predict that floralozone would likely be an agonist, as it meets all the criteria though having two methyl groups instead of one on the alpha carbon. Floralozone most definitely does have a green-vegetable perceptual note.

TL;DR - this issue may be invalid, and the docker may have correctly identified an error in the Mainland data.

**Comment by ssepeq** [comment]
[OR3A1_CINN_A.220713.075329.dock.zip](https://github.com/primaryodors/podock/files/9111622/OR3A1_CINN_A.220713.075329.dock.zip)
